cypress/fixtures/profile.json

```diff
@@ -0,0 +1,5 @@
+{
+  "id": 8739,
+  "name": "Jane",
+  "email": "jane@example.com"
+}
```

Comment:
EOF = this means that you should add one blank line to the end of the file. That's why GitHub shows this icon: 
![image](https://user-images.githubusercontent.com/10209180/126734006-081807a8-bc9b-47eb-89f6-2d04a7dc18bd.png)